libs/slam/async_slam/async_slam_worker.cpp

```
@@ -0,0 +1,282 @@
+
+void AsyncSlam::RunLoopClosureAndOptimization_worker(FrameId frame_id, uint64_t timestamp_ns,
+                                                     const Isometry3T& world_from_rig_guess,
+                                                     const Images& current_images) {
+  // init last_step_telemetry_
+  AsyncSlamLCTelemetry last_step_telemetry;
+  last_step_telemetry.timestamp_ns = timestamp_ns;
+
+  TRACE_EVENT ev = profiler_domain_.trace_event("LC & optimization", profiler_color_);
+
+  bool skip_loop_closure = false;
+  if (!last_loop_closures_stamped_.empty()) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)